test/Riok.Mapperly.Tests/Mapping/IgnoreObsoleteTest.cs

```diff
@@ -0,0 +1,227 @@
+﻿using Riok.Mapperly.Abstractions;
+using Riok.Mapperly.Diagnostics;
+
+namespace Riok.Mapperly.Tests.Mapping;
+
+public class IgnoreObsoleteTest
+{
+    private readonly string _classA = """
+        class A
+        {
+            public int Value { get; set;}
+
+            [Obsolete]
+            public int Ignored { get; set;}
+        }
+        """;
+
+    private readonly string _classB = """
+        class B
+        {
+            public int Value { get; set;}
+
+            [Obsolete]
+            public int Ignored { get; set;}
+        }
+        """;
+
+    [Fact]
+    public void ClassAttributeIgnoreObsoleteNone()
```

Comment:
Add a test case which explicitly maps a obsolete property via `[MapProperty]` which should overwrite the obsolete member strategy.